src/main/resources/spring/camel-context.xml

```diff
@@ -233,10 +232,14 @@
                     <to uri="direct:check-rbac-permissions" />
                     <!-- RBAC check end-->
 
-                    <to uri="direct:to-paginationBean"/>
-                    <to uri="direct:to-sortBean"/>
+                    <setHeader headerName="sort_by_defaults">
```

Comment:
I see a redundant header , as I think we could use something similar : 
<param name="sort_by" type="query" defaultValue="provider,datacenter,cluster,vmName" required="false" description="Define sort field which will be use" />
As the code is already doing a **sort_by_defaults.split**

Reply:
this was replaced by `<param name="sort_by" type="query" defaultValue="provider,datacenter,cluster,vmName"`